client/src/pages/recording-view/components/__tests__/global-properties.test.tsx

```diff
@@ -13,6 +13,8 @@ describe('GlobalProperties component', () => {
   const { getValidMetadata, wrapper: Wrapper } = useAllProviders();
 
   beforeAll(() => {
+    nock.cleanAll();
```

Comment:
should this be beforeEach?

Reply:
Can do there is no need because every test does override the methods it calls this was more to avoid tests from other files fo be getting into this.

Reply:
But changed to beforeeach anyway not a big performance impact